### PR TITLE
Insert msgs

### PR DESCRIPTION
**Pull request by nicpottier**

### CODE REVIEW

File: channel.go [hunk 1]
```diff
@@ -181,13 +181,13 @@ const localTTL = 60
...
-	Address     string      `json:"address"       db:"address"`
-	Country     string      `json:"country"       db:"country"`
-	Config      string      `json:"config"        db:"config"`
+	OrgID       OrgID          `json:"org_id"        db:"org_id"`
+	ID          ChannelID      `json:"id"            db:"id"`
+	UUID        ChannelUUID    `json:"uuid"          db:"uuid"`
+	ChannelType ChannelType    `json:"channel_type"  db:"channel_type"`
+	Address     sql.NullString `json:"address"       db:"address"`
+	Country     sql.NullString `json:"country"       db:"country"`
+	Config      sql.NullString `json:"config"        db:"config"`
```
Comment (rowanseymour) [inline channel.go]: Not so related to this PR, but might be simpler if this was a map and the JSON parsing happened during the DB fetch. It's kind weird that the JSON serialized version of a channel will have serialized JSON as a field

File: courier.toml [hunk 2]
```diff
@@ -15,7 +15,7 @@ base_url = "https://localhost:8080"
 port = 8080
 
 # Our database connection string, right now only Postgres is supported
-db = "postgres://courier@localhost/courier?sslmode=disable"
+db = "postgres://courier@localhost/courier_test?sslmode=disable"
```
Comment (rowanseymour) [inline courier.toml]: No biggie but it was intentional having this not be the db used for unit tests

File: testdata.sql [hunk 3]
```diff
@@ -0,0 +1,28 @@
+
+/* Org with id 100 */
+DELETE FROM orgs_org where id = 100;
```
Comment (rowanseymour) [inline testdata.sql]: Seems odd having explicit deletes in a test data file - shouldn't we just truncate these tables?

File: msg.go [hunk 4]
```diff
@@ -43,6 +38,38 @@ const (
...
+
+// MsgVisibility is the visibility of a message
+type MsgVisibility string
+
+// Possible values for MsgVisibility
+const (
+	MsgVisible       MsgVisibility = "V"
+	MsgDeleted       MsgVisibility = "D"
+	MsgArchived      MsgVisibility = "A"
+	NilMsgVisibility MsgVisibility = ""
```
Comment (rowanseymour) [inline msg.go]: Do we need a nil value - shouldn't new Msgs get the db default like for priority?

Comment (nicpottier) [inline msg.go]: This is Nil at the code level not at the DB level.

File: testdata.sql (hunk 3 above)
Comment (nicpottier) [inline testdata.sql]: Seems like only affecting what matters to us is more friendly

File: courier.toml (hunk 2 above)
Comment (nicpottier) [inline courier.toml]: ya, could go either way here really

**Pull request closed without merging**